ansible/helm_external.yml

```
@@ -24,25 +22,25 @@
       vars:
         external_dir_name: elasticsearch-external
         server_type: elasticsearch
-        network_interface: "{{ elasticsearch_network_interface }}"
+        network_interface: "{{ elasticsearch_network_interface | default('') }}"
```

Defaulting to empty string is strange. If something is not set, we should not instead set it to an empty string: use the `omit` filter to keep `network_interface` undefined, which is easier to work with than empty string variables.

```suggestion
        network_interface: "{{ elasticsearch_network_interface | default(omit) }}"
```

(same for the other cases in this file)